tests/Oryx.RuntimeImage.Tests/TestBase.cs

```diff
@@ -11,12 +11,36 @@ namespace Microsoft.Oryx.RuntimeImage.Tests
 {
     public abstract class TestBase
     {
+        protected const string _imageBaseEnvironmentVariable = "ORYX_TEST_IMAGE_BASE";
+        protected const string _tagSuffixEnvironmentVariable = "ORYX_TEST_TAG_SUFFIX";
+        protected const string _defaultImageBase = "oryxdevmcr.azurecr.io";
```

Comment:
`_registryName`?

Reply:
Changed this to `_defaultRegistryName` -- let me know if this doesn't work.